.github/workflows/lint-and-test.yml

```diff
@@ -32,15 +34,27 @@ jobs:
       run: |
         pip install poetry==1.5.1
         poetry install
-
+    
+    - name: Install mdbtools
+      if: startsWith(matrix.os, 'ubuntu')
+      run: |
+        sudo apt-get update
+        sudo apt-get install mdbtools
+    
     - name: Lint
       run: |
         poetry run black --check veg2hab
         poetry run isort --check veg2hab
 
-    - name: Test with pytest
+    - name: Test with pytest ubuntu
+      if: startsWith(matrix.os, 'ubuntu')
+      run: |
+        poetry run pytest -v tests --run_slow
+
+    - name: Test with pytest windows
+      if: startsWith(matrix.os, 'windows')
       run: |
-        poetry run pytest -v tests
+        poetry run pytest -v tests --run_slow -k "not test_editable_vegtypen and not test_tool_by_tool_walkthrough"
```

Comment:
Kan ik het beter zo doen, of in 1 stap?

Reply:
Kan dit niet in de code worden afgevangen, de test_editable vegtypen ging al automatisch goed.. Vanwege die pytetst.skip

Reply:
Ik vind het netter voelen om het hier te doen dan in de code, dan worden er namelijk nooit stiekem tests over geslagen zonder dat je dat expliciet specifieert. (Zou dan --run-slow misschien ook omgekeerd moeten werken? Dus een --skip-slow option?)

Anders stel ik ieg voor dat we er net zoals bij slow een option van maken zodat dingen overslaan consistent geregeld is.

Reply:
`--run-slow` of `--skip-slow` is mij om het even.